src/lv_objx/lv_spinbox.c

```diff
@@ -116,6 +117,20 @@ lv_obj_t * lv_spinbox_create(lv_obj_t * par, const lv_obj_t * copy)
  * Setter functions
  *====================*/
 
+/**
+ * Set spinbox rollover function
+ * @param spinbox pointer to spinbox
+ * @param b true or false to enable or disable (default)
+ */
+void lv_spinbox_set_rollover(lv_obj_t * spinbox, bool b) {
+    LV_ASSERT_OBJ(spinbox, LV_OBJX_NAME);
+
+    lv_spinbox_ext_t * ext = lv_obj_get_ext_attr(spinbox);
+    if(ext == NULL) return;
```

Comment:
The object won't even be able to draw properly if this is `NULL` so I'm not sure if it's worth checking here.

Reply:
Removed NULL check